ocaml/xapi/xapi_pgpu.ml

```diff
@@ -113,7 +113,6 @@ let is_local_pgpu ~__context (pci_ref, pci_rec) =
   let localhost = Helpers.get_localhost ~__context in
   pci_rec.Db_actions.pCI_host = localhost
   && Xapi_pci.(is_class_of_kind Display_controller (int_of_id (pci_rec.Db_actions.pCI_class_id)))
-  && pci_rec.Db_actions.pCI_physical_function = Ref.null (* Ignore PCIs known to be VFs: we don't want PGPUs for them. *)
```

Comment:
Just for completeness - I've confirmed we won't because mxgpu_set_phys_fn_ref returns true if it detects this PCI device as a VF, so this function will return false and we don't get a pGPU defined.